api/indexer.oas2.json

```diff
@@ -2228,6 +2258,215 @@
+      "type": "object",
+      "properties": {
+        "sig-commit": {
+          "description": "\\[c\\]",
+          "type": "string",
+          "format": "byte"
+        },
+        "signed-weight": {
+          "description": "\\[w\\]",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "sig-proofs": {
+          "description": "\\[S\\]",
+          "$ref": "#/definitions/MerkleArrayProof"
+        },
+        "part-proofs": {
+          "description": "\\[P\\]",
+          "$ref": "#/definitions/MerkleArrayProof"
+        },
+        "salt-version": {
+          "description": "\\[v\\] Salt version of the merkle signature.",
+          "type": "integer"
+        },
+        "reveals": {
+          "type": "array",
+          "description": "\\[r\\] Note that this is actually stored as a map[uint64] - Reveal in the actual msgp",
+          "items": {
+            "$ref": "#/definitions/StateProofReveal"
+          }
+        },
+        "positions-to-reveal": {
+          "description": "\\[pr\\] Sequence of reveal positions.",
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "x-algorand-format": "uint64"
+          }
+        }
+      }
+    },
+    "StateProofMessage": {
+      "type": "object",
+      "properties": {
+        "block-headers-commitment": {
+          "description": "\\[b\\]",
+          "type": "string",
+          "format": "byte"
+        },
+        "voters-commitment": {
+          "description": "\\[v\\]",
+          "type": "string",
+          "format": "byte"
+        },
+        "ln-proven-weight": {
+          "description": "\\[P\\]",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "first-attested-round": {
+          "description": "\\[f\\]",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "latest-attested-round": {
+          "description": "\\[l\\]",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        }
+      }
+    },
+    "StateProofReveal": {
+      "type": "object",
+      "properties": {
+        "position": {
+          "description": "The position in the signature and participants arrays corresponding to this entry.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "sig-slot": {
+          "description": "\\[s\\]",
+          "$ref": "#/definitions/StateProofSigSlot"
+        },
+        "participant": {
+          "description": "\\[p\\]",
+          "$ref": "#/definitions/StateProofParticipant"
+        }
+      }
+    },
+    "StateProofSigSlot": {
+      "type": "object",
+      "properties": {
+        "signature": {
+          "description": "",
+          "$ref": "#/definitions/StateProofSignature"
+        },
+        "lower-sig-weight": {
```

Comment:
nits: I was not able to find an exact name for this field. And I'm not sure this is accurate. can we rename it to `LValue`?

Reply:
I guess it is okay. thanks.